dcos/config.py

```diff
@@ -43,7 +43,16 @@ def set_val(name, value):
 
     save(toml_config)
 
-    if not value_exists:
+    if name == "core.dcos_acs_token":
+        msg = "[{}]: ".format(name)
+        if not value_exists:
+            msg += "set"
+        elif old_value == new_value:
+            msg += "already set to that value"
+        else:
+            msg += "changed"
+        emitter.publish(msg)
+    elif not value_exists:
```

Comment:
I noticed that we were print this value even if I don't run the command `dcos config ...`. Can you verity that we don't print this to `stdout` as it will break scripts.